### ISSUE TITLE
Feedparse and http request nodes error with https on Windows 10

### ISSUE THREAD

**Issue by TotallyInformation** [node-red/node-red-nodes#344]
On Windows 10 latest build with Node.js v6.11.0 and NR v0.17.4, Feedparse fails to connect to a feedburner feed using https:

```
https://feeds.feedburner.com/GizmagEmergingTechnologyMagazine
```

That works fine manually and in other feed readers. The error that is generated is:

```
Error: Hostname/IP doesn't match certificate's altnames: "Host: feeds.feedburner.com. is not in the cert's altnames: DNS:misc.google.com, DNS:*.1hourpersecond.com, DNS:*.acceleratewithgoogle.com, DNS:*.actions.google.com, DNS:*.adgoogle.net, DNS:*.admeld.com, DNS:*.advertiserscommunity.com, DNS:*.adwords-community.com, DNS:*.adwordsexpress.com, DNS:*.agoogleday.com, DNS:*.angulardart.org, DNS:*.api.ai, DNS:*.appenginedemo.com, DNS:*.apture.com, DNS:*.area120.com, DNS:*.artproject.com, DNS:*.baseline.google.com, DNS:*.baselinestudy.com, DNS:*.baselinestudy.org, DNS:*.beatthatquote.com, DNS:*.bertappward.com, DNS:*.blink.org, DNS:*.brotli.org, DNS:*.bumpshare.com, DNS:*.bumptop.ca, DNS:*.bumptop.com, DNS:*.bumptop.net, DNS:*.bumptop.org, DNS:*.bumptunes.com, DNS:*.campuslondon.co.uk, DNS:*.campuslondon.com, DNS:*.campustelaviv.com, DNS:*.certificate-transparency.org, DNS:*.chrome.com, DNS:*.chromebook.com, DNS:*.chromebooks.com, DNS:*.chromecast.com, DNS:*.chromeshorts.com, DNS:*.chromium.org, DNS:*.cloudburstresearch.com, DNS:*.cloudfunctions.net, DNS:*.cloudrobotics.com, DNS:*.conscrypt.com, DNS:*.conscrypt.org, DNS:*.consumerbarometer2013.com, DNS:*.coova.com, DNS:*.coova.net, DNS:*.coova.org, DNS:*.crr.com, DNS:*.cs4hs.com, DNS:*.culturalspot.com, DNS:*.dartsummit.com, DNS:*.data-vocabulary.org, DNS:*.debug.com, DNS:*.debugproject.com, DNS:*.design.google, DNS:*.developer.google.com, DNS:*.developers.google.com, DNS:*.developersguidetoappgalaxy.com, DNS:*.developersguidetotheappgalaxy.com, DNS:*.devguidetoappgalaxy.com, DNS:*.devguidetotheappgalaxy.com, DNS:*.devsitetest.how, DNS:*.digitalworkshop.de, DNS:*.doodle4google.com, DNS:*.environment.google, DNS:*.episodic.com, DNS:*.feedburner.com, DNS:*.fflick.com, DNS:*.fiberforcommunities.com, DNS:*.financeleadsonline.com, DNS:*.firebase.google.com, DNS:*.flutterapp.com, DNS:*.freeandopenweb.com, DNS:*.g-tun.com, DNS:*.galaxynexus.com, DNS:*.gamingyoutube.com, DNS:*.gbbo.co.uk, DNS:*.gbc.beatthatquote.com, DNS:*.gerritcodereview.com, DNS:*.getbumptop.com, DNS:*.geteverybusinessonline.com, DNS:*.getgoogleback.com, DNS:*.getyourgoogleback.com, DNS:*.gipscorp.com, DNS:*.glass-community.com, DNS:*.globaledu.org, DNS:*.goldenhourcamera.com, DNS:*.gonglchuangl.net, DNS:*.google.berlin, DNS:*.google.domains, DNS:*.google.org, DNS:*.google.ventures, DNS:*.googleaday.com, DNS:*.googleanalytics.com, DNS:*.googleapps.com, DNS:*.googleartproject.com, DNS:*.googleartproject.org, DNS:*.googlebrandlab.com, DNS:*.googlecompare.co.uk, DNS:*.googledanmark.com, DNS:*.googledomains.com, DNS:*.googleelections.com, DNS:*.googleenterprise.com, DNS:*.googlefinland.com, DNS:*.googleforveterans.com, DNS:*.googleforwork.com, DNS:*.googleideas.com, DNS:*.googleideas.org, DNS:*.googleinsidesearch.com, DNS:*.googlemaps.com, DNS:*.googlephotos.com, DNS:*.googleplay.com, DNS:*.googleplus.com, DNS:*.googlesverige.com, DNS:*.googletraveladservices.com, DNS:*.googlevendorcontent.com, DNS:*.googleventures.com, DNS:*.gregandgloriaspizza.com, DNS:*.gsrc.io, DNS:*.guidetoappgalaxy.com, DNS:*.guidetotheappgalaxy.com, DNS:*.hindiweb.com, DNS:*.howtogetmo.co.uk, DNS:*.html5rocks.com, DNS:*.hwgo.com, DNS:*.impermium.com, DNS:*.j2objc.org, DNS:*.jacquard.com, DNS:*.jamboard.com, DNS:*.keytransparency.com, DNS:*.keytransparency.foo, DNS:*.keytransparency.org, DNS:*.loonforall.com, DNS:*.madewithcode.com, DNS:*.mdialog.com, DNS:*.messagesforjapan.org, DNS:*.mfg-inspector.com, DNS:*.miraikiroku.com, DNS:*.miraikoroku.com, DNS:*.mobileresearchub.com, DNS:*.mobileview.page, DNS:*.mygbiz.com, DNS:*.near.by, DNS:*.netz-verteidiger.de, DNS:*.netzverteidiger.de, DNS:*.oauthz.com, DNS:*.on.here, DNS:*.on2.com, DNS:*.onetoday.com, DNS:*.onetoday.org, DNS:*.oneworldmanystories.com, DNS:*.ongbiz.com, DNS:*.openhandsetalliance.com, DNS:*.ornek-isletme.com, DNS:*.pagespeedmobilizer.com, DNS:*.partylikeits1986.org, DNS:*.paxlicense.org, DNS:*.personfinder.google.org, DNS:*.photosphere.com, DNS:*.picasa.com, DNS:*.pittpatt.com, DNS:*.polymerproject.org, DNS:*.postini.com, DNS:*.privacychoices.org, DNS:*.projectara.com, DNS:*.projectbaseline.com, DNS:*.projectloon.com, DNS:*.projectsoli.com, DNS:*.questvisual.com, DNS:*.quickoffice.com, DNS:*.quiksee.com, DNS:*.quoteproxy.beatthatquote.com, DNS:*.recaptcha.net, DNS:*.revolv.com, DNS:*.reworkwithgoogle.com, DNS:*.ridepenguin.com, DNS:*.rightlanebikeshop.com, DNS:*.s.svc-1.google.com, DNS:*.saynow.com, DNS:*.schemer.com, DNS:*.screenwiseselect.com, DNS:*.screenwisetrends.com, DNS:*.screenwisetrendspanel.com, DNS:*.shibboleth.tv, DNS:*.snapseed.com, DNS:*.solveforx.com, DNS:*.spider.io, DNS:*.staging.widevine.com, DNS:*.storage-nightly-test.googleusercontent.com, DNS:*.storage-staging-test.googleusercontent.com, DNS:*.storage-test-test.googleusercontent.com, DNS:*.support.google.com, DNS:*.teachparentstech.com, DNS:*.teachparentstech.org, DNS:*.tensorflow.org, DNS:*.thecleversense.com, DNS:*.thedigitalgarage.co.uk, DNS:*.theglasscollective.com, DNS:*.thinkquarterly.co.uk, DNS:*.thinkquarterly.com, DNS:*.toastedbeanscoffee.com, DNS:*.txcloud.net, DNS:*.txvia.com, DNS:*.uat.widevine.com, DNS:*.unternehmen-beispiel.de, DNS:*.useplannr.com, DNS:*.v8project.org, DNS:*.verily.com, DNS:*.verilylifesciences.com, DNS:*.wallet.com, DNS:*.waymo.com, DNS:*.waze.com, DNS:*.webappfieldguide.com, DNS:*.wesolveforx.com, DNS:*.whatbrowser.com, DNS:*.whatbrowser.org, DNS:*.widevine.com, DNS:*.womenwill.com, DNS:*.x.company, DNS:*.x.team, DNS:*.xn--9trs65b.com, DNS:*.youcanstopspam.com, DNS:*.youtubegaming.com, DNS:*.youtubemobilesupport.com, DNS:*.zagat.com, DNS:1hourpersecond.com, DNS:acceleratewithgoogle.com, DNS:adgoogle.net, DNS:admeld.com, DNS:advertisercommunity.com, DNS:advertiserscommunity.com, DNS:adwords-community.com, DNS:adwordsexpress.com, DNS:agoogleday.com, DNS:angulardart.org, DNS:api.ai, DNS:appenginedemo.com, DNS:apture.com, DNS:area120.com, DNS:artproject.com, DNS:baselinestudy.com, DNS:baselinestudy.org, DNS:beatthatquote.com, DNS:bertappward.com, DNS:blink.org, DNS:brotli.org, DNS:bumpshare.com, DNS:bumptop.ca, DNS:bumptop.com, DNS:bumptop.net, DNS:bumptop.org, DNS:bumptunes.com, DNS:campuslondon.co.uk, DNS:campuslondon.com, DNS:campustelaviv.com, DNS:certificate-transparency.org, DNS:chrome.com, DNS:chromebook.com, DNS:chromebooks.com, DNS:chromecast.com, DNS:chromeshorts.com, DNS:chromium.org, DNS:clickserve.dartsearch.net, DNS:clickserve.eu.dartsearch.net, DNS:clickserve.uk.dartsearch.net, DNS:clickserve.us2.dartsearch.net, DNS:clickserver.googleads.com, DNS:cloudburstresearch.com, DNS:cloudfunctions.net, DNS:cloudrobotics.com, DNS:conscrypt.com, DNS:conscrypt.org, DNS:consumerbarometer2013.com, DNS:cookiechoices.org, DNS:coova.com, DNS:coova.net, DNS:coova.org, DNS:crr.com, DNS:cs4hs.com, DNS:culturalspot.com, DNS:dartsummit.com, DNS:data-vocabulary.org, DNS:debug.com, DNS:debugproject.com, DNS:design.google, DNS:developer.dev.nestlabs.com, DNS:developer.ft.nest.com, DNS:developer.integration.nestlabs.com, DNS:developer.nest.com, DNS:developer.qa.nestlabs.com, DNS:developer.stable.nestlabs.com, DNS:developers.dev.nestlabs.com, DNS:developers.ft.nest.com, DNS:developers.integration.nestlabs.com, DNS:developers.nest.com, DNS:developers.qa.nestlabs.com, DNS:developers.stable.nestlabs.com, DNS:developersguidetoappgalaxy.com, DNS:developersguidetotheappgalaxy.com, DNS:devguidetoappgalaxy.com, DNS:devguidetotheappgalaxy.com, DNS:devsitetest.how, DNS:digitalworkshop.de, DNS:doodle4google.com, DNS:environment.google, DNS:episodic.com, DNS:feedburner.com, DNS:fflick.com, DNS:fiberforcommunities.com, DNS:financeleadsonline.com, DNS:flutterapp.com, DNS:freeandopenweb.com, DNS:g-tun.com, DNS:galaxynexus.com, DNS:gamingyoutube.com, DNS:gbbo.co.uk, DNS:gbc.beatthatquote.com, DNS:gerritcodereview.com, DNS:getbumptop.com, DNS:geteverybusinessonline.com, DNS:getgoogleback.com, DNS:getyourgoogleback.com, DNS:gipscorp.com, DNS:glass-community.com, DNS:globaledu.org, DNS:goldenhourcamera.com, DNS:gonglchuangl.net, DNS:goog.dmtry.com, DNS:google.berlin, DNS:google.domains, DNS:google.org, DNS:google.ventures, DNS:googleaday.com, DNS:googleanalytics.com, DNS:googleapps.com, DNS:googleartproject.com, DNS:googleartproject.org, DNS:googlebrandlab.com, DNS:googlecompare.co.uk, DNS:googledanmark.com, DNS:googledomains.com, DNS:googleelections.com, DNS:googleenterprise.com, DNS:googlefinland.com, DNS:googleforveterans.com, DNS:googleforwork.com, DNS:googleideas.com, DNS:googleideas.org, DNS:googleinsidesearch.com, DNS:googlemaps.com, DNS:googlephotos.com, DNS:googleplay.com, DNS:googleplus.com, DNS:googlesverige.com, DNS:googletraveladservices.com, DNS:googlevendorcontent.com, DNS:googleventures.com, DNS:gregandgloriaspizza.com, DNS:gsrc.io, DNS:guidetoappgalaxy.com, DNS:guidetotheappgalaxy.com, DNS:hindiweb.com, DNS:howtogetmo.co.uk, DNS:html5rocks.com, DNS:hwgo.com, DNS:images.zagat.com, DNS:impermium.com, DNS:j2objc.org, DNS:jacquard.com, DNS:jamboard.com, DNS:js.dmtry.com, DNS:keytransparency.com, DNS:keytransparency.foo, DNS:keytransparency.org, DNS:loonforall.com, DNS:madewithcode.com, DNS:mdialog.com, DNS:messagesforjapan.org, DNS:mfg-inspector.com, DNS:miraikiroku.com, DNS:miraikoroku.com, DNS:mobileresearchub.com, DNS:mobileview.page, DNS:mygbiz.com, DNS:n339.asp-cc.com, DNS:near.by, DNS:netz-verteidiger.de, DNS:netzverteidiger.de, DNS:oauthz.com, DNS:on.here, DNS:on2.com, DNS:onetoday.com, DNS:onetoday.org, DNS:oneworldmanystories.com, DNS:ongbiz.com, DNS:openhandsetalliance.com, DNS:ornek-isletme.com, DNS:pagespeedmobilizer.com, DNS:partylikeits1986.org, DNS:paxlicense.org, DNS:photosphere.com, DNS:picasa.com, DNS:ping.feedburner.google.com, DNS:pittpatt.com, DNS:polymerproject.org, DNS:postini.com, DNS:privacychoices.org, DNS:projectara.com, DNS:projectbaseline.com, DNS:projectloon.com, DNS:projectsoli.com, DNS:questvisual.com, DNS:quiksee.com, DNS:quoteproxy.beatthatquote.com, DNS:recaptcha.net, DNS:revolv.com, DNS:reworkwithgoogle.com, DNS:ridepenguin.com, DNS:rightlanebikeshop.com, DNS:rootmusic.bandpage.com, DNS:s.svc-1.google.com, DNS:saynow.com, DNS:schemer.com, DNS:screenwiseselect.com, DNS:screenwisetrends.com, DNS:screenwisetrendspanel.com, DNS:snapseed.com, DNS:solveforx.com, DNS:spider.io, DNS:teachparentstech.com, DNS:teachparentstech.org, DNS:tensorflow.org, DNS:thecleversense.com, DNS:thedigitalgarage.co.uk, DNS:theglasscollective.com, DNS:thinkquarterly.co.uk, DNS:thinkquarterly.com, DNS:toastedbeanscoffee.com, DNS:txcloud.net, DNS:txvia.com, DNS:unternehmen-beispiel.de, DNS:useplannr.com, DNS:v8project.org, DNS:verily.com, DNS:verilylifesciences.com, DNS:wallet.com, DNS:waymo.com, DNS:waze.com, DNS:webappfieldguide.com, DNS:weltweitwachsen.de, DNS:wesolveforx.com, DNS:whatbrowser.com, DNS:whatbrowser.org, DNS:womenwill.com, DNS:www.advertisercommunity.com, DNS:www.bandpage.com, DNS:www.cookiechoices.org, DNS:www.weltweitwachsen.de, DNS:x.company, DNS:x.team, DNS:xn--9trs65b.com, DNS:youcanstopspam.com, DNS:youtubegaming.com, DNS:youtubemobilesupport.com, DNS:zagat.com"
```

On the console:

```
16 Jul 11:55:42 - [error] [feedparse:GizMag/NewAtlas] Error: Hostname/IP doesn't match certificate's altnames: "Host: feeds.feedburner.com. is not in the cert's altnames: DNS:misc.google.com, DNS:*.1hourpersecond.com, DNS:*.acceleratewithgoogle.com, DNS:*.actions.google.com, DNS:*.adgoogle.net, DNS:*.admeld.com, DNS:*.advertiserscommunity.com, DNS:*.adwords-community.com, DNS:*.adwordsexpress.com, DNS:*.agoogleday.com, DNS:*.angulardart.org, DNS:*.api.ai, DNS:*.appenginedemo.com, DNS:*.apture.com, DNS:*.area120.com, DNS:*.artproject.com, DNS:*.baseline.google.com, DNS:*.baselinestudy.com, DNS:*.baselinestudy.org, DNS:*.beatthatquote.com, DNS:*.bertappward.com, DNS:*.blink.org, DNS:*.brotli.org, DNS:*.bumpshare.com, DNS:*.bumptop.ca, DNS:*.bumptop.com, DNS:*.bumptop.net, DNS:*.bumptop.org, DNS:*.bumptunes.com, DNS:*.campuslondon.co.uk, DNS:*.campuslondon.com, DNS:*.campustelaviv.com, DNS:*.certificate-transparency.org, DNS:*.chrome.com, DNS:*.chromebook.com, DNS:*.chromebooks.com, DNS:*.chromecast.com, DNS:*.chromeshorts.com, DNS:*.chromium.org, DNS:*.cloudburstresearch.com, DNS:*.cloudfunctions.net, DNS:*.cloudrobotics.com, DNS:*.conscrypt.com, DNS:*.conscrypt.org, DNS:*.consumerbarometer2013.com, DNS:*.coova.com, DNS:*.coova.net, DNS:*.coova.org, DNS:*.crr.com, DNS:*.cs4hs.com, DNS:*.culturalspot.com, DNS:*.dartsummit.com, DNS:*.data-vocabulary.org, DNS:*.debug.com, DNS:*.debugproject.com, DNS:*.design.google, DNS:*.developer.google.com, DNS:*.developers.google.com, DNS:*.developersguidetoappgalaxy.com, DNS:*.developersguidetotheappgalaxy.com, DNS:*.devguidetoappgalaxy.com, DNS:*.devguidetotheappgalaxy.com, DNS:*.devsitetest.how, DNS:*.digitalworkshop.de, DNS:*.doodle4google.com, DNS:*.environment.google, DNS:*.episodic.com, DNS:*.feedburner.com, DNS:*.fflick.com, DNS:*.fiberforcommunities.com, DNS:*.financeleadsonline.com, DNS:*.firebase.google.com, DNS:*.flutterapp.com, DNS:*.freeandopenweb.com, DNS:*.g-tun.com, DNS:*.galaxynexus.com, DNS:*.gamingyoutube.com, DNS:*.gbbo.co.uk, DNS:*.gbc.beatthatquote.com, DNS:*.gerritcodereview.com, DNS:*.getbumptop.com, DNS:*.geteverybusinessonline.com, DNS:*.getgoogleback.com, DNS:*.getyourgoogleback.com, DNS:*.gipscorp.com, DNS:*.glass-community.com, DNS:*.globaledu.org, DNS:*.goldenhourcamera.com, DNS:*.gonglchuangl.net, DNS:*.google.berlin, DNS:*.google.domains, DNS:*.google.org, DNS:*.google.ventures, DNS:*.googleaday.com, DNS:*.googleanalytics.com, DNS:*.googleapps.com, DNS:*.googleartproject.com, DNS:*.googleartproject.org, DNS:*.googlebrandlab.com, DNS:*.googlecompare.co.uk, DNS:*.googledanmark.com, DNS:*.googledomains.com, DNS:*.googleelections.com, DNS:*.googleenterprise.com, DNS:*.googlefinland.com, DNS:*.googleforveterans.com, DNS:*.googleforwork.com, DNS:*.googleideas.com, DNS:*.googleideas.org, DNS:*.googleinsidesearch.com, DNS:*.googlemaps.com, DNS:*.googlephotos.com, DNS:*.googleplay.com, DNS:*.googleplus.com, DNS:*.googlesverige.com, DNS:*.googletraveladservices.com, DNS:*.googlevendorcontent.com, DNS:*.googleventures.com, DNS:*.gregandgloriaspizza.com, DNS:*.gsrc.io, DNS:*.guidetoappgalaxy.com, DNS:*.guidetotheappgalaxy.com, DNS:*.hindiweb.com, DNS:*.howtogetmo.co.uk, DNS:*.html5rocks.com, DNS:*.hwgo.com, DNS:*.impermium.com, DNS:*.j2objc.org, DNS:*.jacquard.com, DNS:*.jamboard.com, DNS:*.keytransparency.com, DNS:*.keytransparency.foo, DNS:*.keytransparency.org, DNS:*.loonforall.com, DNS:*.madewithcode.com, DNS:*.mdialog.com, DNS:*.messagesforjapan.org, DNS:*.mfg-inspector.com, DNS:*.miraikiroku.com, DNS:*.miraikoroku.com, DNS:*.mobileresearchub.com, DNS:*.mobileview.page, DNS:*.mygbiz.com, DNS:*.near.by, DNS:*.netz-verteidiger.de, DNS:*.netzverteidiger.de, DNS:*.oauthz.com, DNS:*.on.here, DNS:*.on2.com, DNS:*.onetoday.com, DNS:*.onetoday.org, DNS:*.oneworldmanystories.com, DNS:*.ongbiz.com, DNS:*.openhandsetalliance.com, DNS:*.ornek-isletme.com, DNS:*.pagespeedmobilizer.com, DNS:*.partylikeits1986.org, DNS:*.paxlicense.org, DNS:*.personfinder.google.org, DNS:*.photosphere.com, DNS:*.picasa.com, DNS:*.pittpatt.com, DNS:*.polymerproject.org, DNS:*.postini.com, DNS:*.privacychoices.org, DNS:*.projectara.com, DNS:*.projectbaseline.com, DNS:*.projectloon.com, DNS:*.projectsoli.com, DNS:*.questvisual.com, DNS:*.quickoffice.com, DNS:*.quiksee.com, DNS:*.quoteproxy.beatthatquote.com, DNS:*.recaptcha.net, DNS:*.revolv.com, DNS:*.reworkwithgoogle.com, DNS:*.ridepenguin.com, DNS:*.rightlanebikeshop.com, DNS:*.s.svc-1.google.com, DNS:*.saynow.com, DNS:*.schemer.com, DNS:*.screenwiseselect.com, DNS:*.screenwisetrends.com, DNS:*.screenwisetrendspanel.com, DNS:*.shibboleth.tv, DNS:*.snapseed.com, DNS:*.solveforx.com, DNS:*.spider.io, DNS:*.staging.widevine.com, DNS:*.storage-nightly-test.googleusercontent.com, DNS:*.storage-staging-test.googleusercontent.com, DNS:*.storage-test-test.googleusercontent.com, DNS:*.support.google.com, DNS:*.teachparentstech.com, DNS:*.teachparentstech.org, DNS:*.tensorflow.org, DNS:*.thecleversense.com, DNS:*.thedigitalgarage.co.uk, DNS:*.theglasscollective.com, DNS:*.thinkquarterly.co.uk, DNS:*.thinkquarterly.com, DNS:*.toastedbeanscoffee.com, DNS:*.txcloud.net, DNS:*.txvia.com, DNS:*.uat.widevine.com, DNS:*.unternehmen-beispiel.de, DNS:*.useplannr.com, DNS:*.v8project.org, DNS:*.verily.com, DNS:*.verilylifesciences.com, DNS:*.wallet.com, DNS:*.waymo.com, DNS:*.waze.com, DNS:*.webappfieldguide.com, DNS:*.wesolveforx.com, DNS:*.whatbrowser.com, DNS:*.whatbrowser.org, DNS:*.widevine.com, DNS:*.womenwill.com, DNS:*.x.company, DNS:*.x.team, DNS:*.xn--9trs65b.com, DNS:*.youcanstopspam.com, DNS:*.youtubegaming.com, DNS:*.youtubemobilesupport.com, DNS:*.zagat.com, DNS:1hourpersecond.com, DNS:acceleratewithgoogle.com, DNS:adgoogle.net, DNS:admeld.com, DNS:advertisercommunity.com, DNS:advertiserscommunity.com, DNS:adwords-community.com, DNS:adwordsexpress.com, DNS:agoogleday.com, DNS:angulardart.org, DNS:api.ai, DNS:appenginedemo.com, DNS:apture.com, DNS:area120.com, DNS:artproject.com, DNS:baselinestudy.com, DNS:baselinestudy.org, DNS:beatthatquote.com, DNS:bertappward.com, DNS:blink.org, DNS:brotli.org, DNS:bumpshare.com, DNS:bumptop.ca, DNS:bumptop.com, DNS:bumptop.net, DNS:bumptop.org, DNS:bumptunes.com, DNS:campuslondon.co.uk, DNS:campuslondon.com, DNS:campustelaviv.com, DNS:certificate-transparency.org, DNS:chrome.com, DNS:chromebook.com, DNS:chromebooks.com, DNS:chromecast.com, DNS:chromeshorts.com, DNS:chromium.org, DNS:clickserve.dartsearch.net, DNS:clickserve.eu.dartsearch.net, DNS:clickserve.uk.dartsearch.net, DNS:clickserve.us2.dartsearch.net, DNS:clickserver.googleads.com, DNS:cloudburstresearch.com, DNS:cloudfunctions.net, DNS:cloudrobotics.com, DNS:conscrypt.com, DNS:conscrypt.org, DNS:consumerbarometer2013.com, DNS:cookiechoices.org, DNS:coova.com, DNS:coova.net, DNS:coova.org, DNS:crr.com, DNS:cs4hs.com, DNS:culturalspot.com, DNS:dartsummit.com, DNS:data-vocabulary.org, DNS:debug.com, DNS:debugproject.com, DNS:design.google, DNS:developer.dev.nestlabs.com, DNS:developer.ft.nest.com, DNS:developer.integration.nestlabs.com, DNS:developer.nest.com, DNS:developer.qa.nestlabs.com, DNS:developer.stable.nestlabs.com, DNS:developers.dev.nestlabs.com, DNS:developers.ft.nest.com, DNS:developers.integration.nestlabs.com, DNS:developers.nest.com, DNS:developers.qa.nestlabs.com, DNS:developers.stable.nestlabs.com, DNS:developersguidetoappgalaxy.com, DNS:developersguidetotheappgalaxy.com, DNS:devguidetoappgalaxy.com, DNS:devguidetotheappgalaxy.com, DNS:devsitetest.how, DNS:digitalworkshop.de, DNS:doodle4google.com, DNS:environment.google, DNS:episodic.com, DNS:feedburner.com, DNS:fflick.com, DNS:fiberforcommunities.com, DNS:financeleadsonline.com, DNS:flutterapp.com, DNS:freeandopenweb.com, DNS:g-tun.com, DNS:galaxynexus.com, DNS:gamingyoutube.com, DNS:gbbo.co.uk, DNS:gbc.beatthatquote.com, DNS:gerritcodereview.com, DNS:getbumptop.com, DNS:geteverybusinessonline.com, DNS:getgoogleback.com, DNS:getyourgoogleback.com, DNS:gipscorp.com, DNS:glass-community.com, DNS:globaledu.org, DNS:goldenhourcamera.com, DNS:gonglchuangl.net, DNS:goog.dmtry.com, DNS:google.berlin, DNS:google.domains, DNS:google.org, DNS:google.ventures, DNS:googleaday.com, DNS:googleanalytics.com, DNS:googleapps.com, DNS:googleartproject.com, DNS:googleartproject.org, DNS:googlebrandlab.com, DNS:googlecompare.co.uk, DNS:googledanmark.com, DNS:googledomains.com, DNS:googleelections.com, DNS:googleenterprise.com, DNS:googlefinland.com, DNS:googleforveterans.com, DNS:googleforwork.com, DNS:googleideas.com, DNS:googleideas.org, DNS:googleinsidesearch.com, DNS:googlemaps.com, DNS:googlephotos.com, DNS:googleplay.com, DNS:googleplus.com, DNS:googlesverige.com, DNS:googletraveladservices.com, DNS:googlevendorcontent.com, DNS:googleventures.com, DNS:gregandgloriaspizza.com, DNS:gsrc.io, DNS:guidetoappgalaxy.com, DNS:guidetotheappgalaxy.com, DNS:hindiweb.com, DNS:howtogetmo.co.uk, DNS:html5rocks.com, DNS:hwgo.com, DNS:images.zagat.com, DNS:impermium.com, DNS:j2objc.org, DNS:jacquard.com, DNS:jamboard.com, DNS:js.dmtry.com, DNS:keytransparency.com, DNS:keytransparency.foo, DNS:keytransparency.org, DNS:loonforall.com, DNS:madewithcode.com, DNS:mdialog.com, DNS:messagesforjapan.org, DNS:mfg-inspector.com, DNS:miraikiroku.com, DNS:miraikoroku.com, DNS:mobileresearchub.com, DNS:mobileview.page, DNS:mygbiz.com, DNS:n339.asp-cc.com, DNS:near.by, DNS:netz-verteidiger.de, DNS:netzverteidiger.de, DNS:oauthz.com, DNS:on.here, DNS:on2.com, DNS:onetoday.com, DNS:onetoday.org, DNS:oneworldmanystories.com, DNS:ongbiz.com, DNS:openhandsetalliance.com, DNS:ornek-isletme.com, DNS:pagespeedmobilizer.com, DNS:partylikeits1986.org, DNS:paxlicense.org, DNS:photosphere.com, DNS:picasa.com, DNS:ping.feedburner.google.com, DNS:pittpatt.com, DNS:polymerproject.org, DNS:postini.com, DNS:privacychoices.org, DNS:projectara.com, DNS:projectbaseline.com, DNS:projectloon.com, DNS:projectsoli.com, DNS:questvisual.com, DNS:quiksee.com, DNS:quoteproxy.beatthatquote.com, DNS:recaptcha.net, DNS:revolv.com, DNS:reworkwithgoogle.com, DNS:ridepenguin.com, DNS:rightlanebikeshop.com, DNS:rootmusic.bandpage.com, DNS:s.svc-1.google.com, DNS:saynow.com, DNS:schemer.com, DNS:screenwiseselect.com, DNS:screenwisetrends.com, DNS:screenwisetrendspanel.com, DNS:snapseed.com, DNS:solveforx.com, DNS:spider.io, DNS:teachparentstech.com, DNS:teachparentstech.org, DNS:tensorflow.org, DNS:thecleversense.com, DNS:thedigitalgarage.co.uk, DNS:theglasscollective.com, DNS:thinkquarterly.co.uk, DNS:thinkquarterly.com, DNS:toastedbeanscoffee.com, DNS:txcloud.net, DNS:txvia.com, DNS:unternehmen-beispiel.de, DNS:useplannr.com, DNS:v8project.org, DNS:verily.com, DNS:verilylifesciences.com, DNS:wallet.com, DNS:waymo.com, DNS:waze.com, DNS:webappfieldguide.com, DNS:weltweitwachsen.de, DNS:wesolveforx.com, DNS:whatbrowser.com, DNS:whatbrowser.org, DNS:womenwill.com, DNS:www.advertisercommunity.com, DNS:www.bandpage.com, DNS:www.cookiechoices.org, DNS:www.weltweitwachsen.de, DNS:x.company, DNS:x.team, DNS:xn--9trs65b.com, DNS:youcanstopspam.com, DNS:youtubegaming.com, DNS:youtubemobilesupport.com, DNS:zagat.com"
```

This appears to be another example of the wildcard altnames issue that has been seen with one or two other nodes.

The HTTP request node does the same thing with any endpoint that uses wildcard certificates that browsers consider valid - e.g. my own blog at https://it.knightnet.org.uk - which uses CloudFlare.

I thought it might be an upstream issue but the following node.js code works just fine:

```javascript
var https = require('https');

var options = {
    host: 'feeds.feedburner.com',
    port: 443,
    path: '/GizmagEmergingTechnologyMagazine',
    method: 'GET',
    headers: {
        accept: '*/*'
    }
};

var req = https.request(options, function (res) {
    console.log(res.statusCode);
    res.on('data', function (d) {
        process.stdout.write(d);
    });
});
req.end();

req.on('error', function (e) {
    console.error(e);
});
```

**Comment by TotallyInformation** [comment]
This is still happening - anyone any ideas?

It is preventing me from developing a solution for aggregating feeds.

**Comment by hardillb** [comment]
Worth testing with the follow-redirects node in play as that's what the httprequest node actually uses under the covers. (https://github.com/node-red/node-red/blob/29cae9975ec27a8b4f2e84a3556ce287e507fdf9/nodes/core/io/21-httprequest.js#L20)

```var https = require("follow-redirects").https;```

It's all working properly on Linux, so I can only guess that it's something that's being handed off to some Windows system library for some reason.

**Comment by TotallyInformation** [comment]
Thanks. I'm bored with defining glossary entries and requirements for my work programme so I'm going for some light relief by doing some hacking of the node to see if I can work it out. I will check your suggestion. It used to work but stopped sometime around the start of the year I think. Just never got round to trying to fix it. I suspect the http request node has the same problem. Again though, only on Windows 10 which is strange.

**Comment by TotallyInformation** [comment]
I've not made much progress but I have discovered that it is a problem with using the "request" module https://www.npmjs.com/package/request

The strange thing is that if I create a simple test js file to call that module (it is in the same folder as the node so I know for sure I'm using the same version, it works absolutely fine. But when called from within Node-RED, it fails.

I've also found out that it is the request module and not the feedparse module that the error is coming from. About line 29 of 32-feedparse.js I think.

But I've not been able to find what is generating the error in the request module. As I say, it works fine if I do this:

```
var request = require("request");

var req = request('https://www.techdirt.com/techdirt_rss.xml', {timeout: 10000, pool: false});

req.on('error', function (e) {
    console.error(e);
});

req.on('response', function (res) {
    console.log(res.statusCode);
    console.dir(res.headers);
});
```

That is pretty much the same code as in 32-feedparse.js so I can only think that it is something environmental to Node-RED that causes it.

My knowledge of JavaScript debugging is about at it's limit now so I'll have to give up.

I have 2 feeds that work and 1 that doesn't. I'm stuck now.
